tv.kodi.Kodi.appdata.xml

```diff
@@ -29,7 +30,6 @@
         and the internet. Our forums and Wiki are bursting with knowledge
         and help for the new user right up to the application developer.</p>
     </description>
-    <launchable type="desktop-id">tv.kodi.Kodi.desktop</launchable>
```

Comment:
You didn't fix it, you hid the problem. You need a launchable.

Reply:
Looking at other appdata files on github.com/flathub, I could not find any quickly that even uses `launchable` at all.

Reply:
I will put this back though. still works with it.

Reply:
If your `id` is `foo.desktop` it works without `launchable` but that is the legacy naming scheme.